src/tests/movieCount.test.js

```diff
@@ -0,0 +1,15 @@
+import updateMovieCardCount from '../modules/movieCount.js';
+
+describe('updateMovieCardCount', () => {
+  test('should update the movie card count', () => {
+    const movieCardCount = document.createElement('span');
+    movieCardCount.id = 'span';
+    document.body.appendChild(movieCardCount);
+
+    updateMovieCardCount(5);
+
+    expect(movieCardCount.textContent).toEqual('5');
+
+    document.body.removeChild(movieCardCount);
+  });
```

Comment:
To make your test more robust and efficient, I recommend tweaking the tests also to check for a case where there is no movie item in the dom.